### PR TITLE
update docs for the polyglot notebooks to include using venvs

### PR DESCRIPTION
**Pull request by shibbas**
Update the documentation for python to include how to use with virtual environments. This talks about both the new `--conda-env` option and adding conda-envs and venvs to Jupyter kernel spec.

### CODE REVIEW

File: docs/jupyter-in-polyglot-notebooks.md [hunk 1]
```diff
@@ -6,15 +6,22 @@ This feature is currently in preview.
...
+3. If you are working with R - Install [R](https://cran.r-project.org/)
 
 ## Connecting to a Python kernel
 Run the following command in a notebook cell:
+If working with Anaconda:
 ```
 #!connect jupyter --kernel-name pythonkernel --conda-env base --kernel-spec python3
 ```
 
+If working with Python and Jupyter directly:
```
Comment (sharoncxu) [inline docs/jupyter-in-polyglot-notebooks.md:20]: What does it mean to be working with Python and Jupyter directly as opposed to with Anaconda?

File: docs/jupyter-in-polyglot-notebooks.md [hunk 2]
```diff
@@ -28,10 +35,15 @@ install.packages('IRkernel')
 IRkernel::installspec() 
 ```
 
-Then restart VS Code from the Anaconda prompt, and run the following command in a notebook cell:
+You might need to restart VSCode.
```
Comment (sharoncxu) [inline docs/jupyter-in-polyglot-notebooks.md:38]: In what cases will the user need to restart VS Code?

File: docs/jupyter-in-polyglot-notebooks.md (hunk 1 above)
Comment (shibbas) [reply docs/jupyter-in-polyglot-notebooks.md:20]: Line 2 above in Setup. You can install Python on your machine and add it to Path and then pip install Jupyter.

File: docs/jupyter-in-polyglot-notebooks.md (hunk 2 above)
Comment (shibbas) [reply docs/jupyter-in-polyglot-notebooks.md:38]: To be able to retrieve the kernelspecs that were newly added or if you add the environment variables new.

File: docs/jupyter-in-polyglot-notebooks.md (hunk 1 above)
Comment (sharoncxu) [reply docs/jupyter-in-polyglot-notebooks.md:20]: Got it. can we change the verbiage to;

"If working with Jupyter using Anaconda"

and:

"If you are working with Python and Jupyter directly without Anaconda" 

to make it more explicit

File: docs/jupyter-in-polyglot-notebooks.md (hunk 2 above)
Comment (sharoncxu) [reply docs/jupyter-in-polyglot-notebooks.md:38]: Could we add this to the explanation?

E.g. "If you installed the new kernelspec or added new environment variables, you will need to restart VS Code"

**Review by sharoncxu — APPROVED** [review]

File: docs/jupyter-in-polyglot-notebooks.md (hunk 2 above)
Comment (shibbas) [reply docs/jupyter-in-polyglot-notebooks.md]: Done!

File: docs/jupyter-in-polyglot-notebooks.md (hunk 1 above)
Comment (shibbas) [reply docs/jupyter-in-polyglot-notebooks.md]: Done!

**Review by AbhitejJohn — APPROVED** [review]

**Pull request merged**